### PR TITLE
R workflowscriptscommon

### PR DESCRIPTION
**Pull request by pinin4fjords**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

This PR adds a recipe to install an R package which will support wrappers around the functions of certain R packages, specifically in the realm of single-cell RNA-seq analysis. Recipes providing those wrapper scripts will be the subject of future PRs, and are dependent upon this recipe.

### CODE REVIEW

File: recipes/r-workflowscriptscommon/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+    build:
```
Comment (bgruening) [inline recipes/r-workflowscriptscommon/meta.yaml]: This should be `host` I think. Is `git` really needed during the build-phase?

Comment (pinin4fjords) [reply recipes/r-workflowscriptscommon/meta.yaml]: Yep, sorry, Git was left over from earlier development. Have fixed and changed to host.

**Comment by bgruening** [comment]
Thanks @pinin4fjords!

**Comment by pinin4fjords** [comment]
Thanks @bgruening for the lightning review! Can I be a contributor? I asked nicely at https://github.com/bioconda/bioconda-recipes/issues/1 :-)

**Comment by bgruening** [comment]
On it, just give me a sec :)

**Comment by pinin4fjords** [comment]
Awesome- thanks for the quick work.

**Pull request merged**